src/main/drivers/pwm_output.c

```diff
@@ -380,3 +385,41 @@ void servoDevInit(const servoDevConfig_t *servoConfig)
 }
 
 #endif
+
+
+#ifdef BEEPER
+void pwmWriteBeeper(bool onoffBeep)
+{
+        if(onoffBeep == true){
+            *beeperPwm.ccr = (1000000/freqBeep)/2;
+            beeperPwm.enabled = true;
+        } else {
+            *beeperPwm.ccr = 0;
+            beeperPwm.enabled = false;
+        }
+}
+
+void pwmToggleBeeper(void)
+{
+        pwmWriteBeeper(!beeperPwm.enabled);
+}
+
+void beeperPwmInit(uint16_t frequency)
```

Comment:
I added the GetByTag checks.

In beeperPwmInit, the function pwmOutConfig is needed and is only accessible inside pwm_output.c. 
Am I missing something?